app/assets/stylesheets/logixRelease.scss

```diff
@@ -271,6 +271,10 @@ a.btn.btn-outline-danger {
   padding-top: 40px;
 }
 
+.col-lg-6{
```

Comment:
Do not override core bootstrap classes

Reply:
shall I add new class then? because there are only bootstrap classes in that div

Reply:
okay so i have applied ```  align-items: center;``` to the class ```.row-about```  instead of the bootstrap classes which targets these elements.

Reply:
Don't change the row-about class, it is getting used at a lot of places which may not go well with this change. Just add a custom class to the element you want to change and then add the styles to it. @abstrekt

Reply:
ok got it @aayush-05

Reply:
please review, I added a custom class ```.benefits``` wherever required. @aayush-05